scylla/src/frame/response/result.rs

```diff
@@ -242,6 +253,12 @@ impl CQLValue {
         }
     }
 
+    pub fn as_varint(self) -> Option<i64> {
```

Comment:
`varint` is not an 8-byte type. It can has arbitrary length, as stated in the protocol documentation:
```
6.23 varint

  A variable-length two's complement encoding of a signed integer.

  The following examples may help implementors of this spec:

  Value | Encoding
  ------|---------
      0 |     0x00
      1 |     0x01
    127 |     0x7F
    128 |   0x0080
    129 |   0x0081
     -1 |     0xFF
   -128 |     0x80
   -129 |   0xFF7F

  Note that positive numbers must use a most-significant byte with a value
  less than 0x80, because a most-significant bit of 1 indicates a negative
  value.  Implementors should pad positive values that have a MSB >= 0x80
  with a leading 0x00 byte.
```
It should be represented either as a big integer from some crate (I saw num-bigint as an example), or at least as raw vector of bytes.
You can take a look at cdrs's implementation for a Rust reference: https://github.com/AlexPikalov/cdrs/blob/42c048fc47b123c46a9260f1d224128c7e8ab530/src/types/data_serialization_types.rs#L184-L186